lib/util/dom_utils.js

```diff
@@ -78,6 +78,16 @@ shaka.util.Dom = class {
     return shaka.util.Dom.asHTMLElement(elements[0]);
   }
 
+  /**
+   * Returns the element with a given class name.
```

Comment:
This seems unnecessary.  Why not call it directly?

Reply:
This is removed.